src/main/java/teammates/ui/controller/InstructorEditInstructorFeedbackSaveAction.java

```diff
@@ -211,4 +212,15 @@ protected RedirectResult createSpecificRedirectResult() {
 
         return result;
     }
+
+    @Override
+    protected void appendToStatusToAdmin(FeedbackResponseCommentAttributes feedbackResponseComment) {
```

Comment:
This method may still require some renaming to indicate that this method involves a comment, as it feels a little out of place to have a general-sounding method for feedback in general to target only comments.